### ISSUE TITLE
Non-module aware go: unknown reference to /v3 package

### ISSUE THREAD

**Issue by cdepauw** [mediocregopher/radix#141]
**Go version:**
go version go1.12 windows/amd64

**Go get method:**
go get github.com/mediocregopher/radix

--> go get for /v3 fails with:
package github.com/mediocregopher/radix/v3: cannot find package "github.com/mediocregopher/radix/v3" in any of:
// $GOROOT
// $GOPATH

**Issue:**
Using the library functions fine. Vendoring (godep) fails, as there are references to /v3 which cannot be resolved.

**Comment by cdepauw** [comment]
I just noticed I completely missed https://github.com/mediocregopher/radix/issues/53 which is essentially dealing with the same topic. Closing this issue due to this.

The local fix applied now was a text removal of all "/v3" in your repo and doing a local commit. That satisfied godep and allowed me to vendor it. Only downside is that other devs pitching in on my repo will need to apply the same trick if they want/need to revendor.
